xfce-base/xfce4-settings/files/xfce4-settings-4.15.1-install-icons.patch

```diff
@@ -0,0 +1,70 @@
+# Upstream issue: https://gitlab.xfce.org/xfce/xfce4-settings/-/issues/176
+# This issue has been fixed in upstream master, so this patch won't be need in the future.
+
+--- ./Makefile.am	2020-05-27 02:54:17.000000000 +0200
++++ /home/atom/local/xfce4-settings/./Makefile.am	2020-05-30 23:53:57.647209221 +0200
```

Comment:
Are you sure you want to leak your local paths? Generally taking the `.patch` off GitLab is preferable.

Reply:
I have removed the local paths. About taking the patch off GitLab: It isn't possible in this case because GitLab doesn't contain `Makefile.in` while the `.tar.bz2` in Gentoo's distfiles does.